gradle.properties

```
@@ -1,4 +1,4 @@
 org.gradle.caching=true
 group=io.ballerinax
 version=7.2.0
-ballerinaLangVersion=2201.4.0-20230124-005500-f19b94e2
+ballerinaLangVersion=2201.7.0-20230619-175900-bb4e4544
```

Ballerina Lang has released the stable version - `2201.7.0`. We can use that here.
```suggestion
ballerinaLangVersion=2201.7.0
```